pants.toml

```diff
@@ -44,6 +44,12 @@ pants_ignore.add = [
   "!/pants.pex",
 ]
 
+build_ignore.add = [
+  # Disable Go targets because the Go backend is experimental (as it is under active development),
+  # and thus is not enabled by default.
+  "/testprojects/src/go/**",
```

Comment:
We're trying to actively kill testprojects to instead have the code live entirely in the relevant integration test using `setup_tmpdir()`. It looks like you aren't using this project in a test, only as a demo. I think it'd probably be best to remove the testproject imo.

--

Why remove?

1. Users have reported looking at pantsbuild/pants for examples of how to structure their repo. The setup in testproject usually is not designed for instruction, but for our testing. We would rather have something like an `example-go` repo for the sake of instruction.
2. The testprojects were brittle - multiple tests often depended on the same project, making it hard to change a test. It required jumping around to multiple files to understand what a test did.

Reply:
> We're trying to actively kill testprojects to instead have the code live entirely in the relevant integration test using setup_tmpdir(). It looks like you aren't using this project in a test, only as a demo. I think it'd probably be best to remove the testproject imo.

Because I found value in being able to run pants against these Go targets even with integration testing. Where I can put a target like that?

Reply:
Okay. Possibly under the backend itself, but this is probably the best spot for now. Thanks